Build/Projects/MonoGame.Framework.definition

```diff
@@ -44,6 +43,7 @@
     <CustomDefinitions>
       <Platform Name="Android">TRACE;ANDROID;GLES;OPENGL;OPENAL</Platform>
       <Platform Name="iOS">IOS;GLES;OPENGL;OPENAL</Platform>
+      <Platform Name="tvOS">IOS;__TVOS__;GLES;OPENGL;OPENAL</Platform>
```

Comment:
Symbols with a preceding double underscore are generally reserved for
system implementations in the C++ world, and not standard in any C#
implementations.

It doesn't hurt to have IOS and ANDROID defined in our projects, even if
Xamarin might declare them as well (is there any documentation to that
effect?), and having TVOS is fine for those parts that need differentiation
at compile time rather than run time.